0-bootstrap/variables.tf

```
@@ -88,6 +88,17 @@ variable "bucket_tfstate_kms_force_destroy" {
   default     = false
 }
 
+variable "vpc" {
+  description = "The type of VPC to attach the project to. Possible options are base or restricted."
+  type        = string
+  default     = "none"
+
+  validation {
+    condition     = var.vpc == "base" || var.vpc == "restricted" || var.vpc == "none"
```

nit: you might consider using [contains ](https://developer.hashicorp.com/terraform/language/functions/contains) - more concise if additional options are added in the future.

```suggestion
    condition     = contains(["none", "base", "restricted"], var.vpc)
```